Helper.cs

```diff
@@ -0,0 +1,64 @@
+﻿
+public class Helper
```

Comment:
Dar pastebejau toki dalyka - is pavadinimo, visiskai neaisku, ka sita klase daro. Siulyciau keisti pavadinima i, pvz., ``` CharacterFilterService ``` ar pan.